app/views/trainees/programme_details/edit.html.erb

```diff
@@ -22,31 +22,36 @@
 
       <%= f.govuk_radio_buttons_fieldset(:main_age_range, legend: { text: "Age range", size: "s" }, classes: "age_range") do %>
         <% main_age_ranges_options.each_with_index do |age_range, index| %>
-          <%= f.govuk_radio_button(:main_age_range, age_range, 
-                                   label: { text: age_range }, 
+          <%= f.govuk_radio_button(:main_age_range, age_range,
+                                   label: { text: age_range },
                                    link_errors: index.zero? ) %>
         <% end %>
 
         <%= f.govuk_radio_divider %>
 
-        <%= f.govuk_radio_button(:main_age_range, 
-                                 :other, 
+        <%= f.govuk_radio_button(:main_age_range,
+                                 :other,
                                  label: { text: "Other age range"} ) do %>
           <%= render_component FormComponents::Autocomplete::View.new(
             form_field: f.govuk_collection_select(:additional_age_range, additional_age_ranges_options,
-                                                  :name, 
-                                                  :name, 
+                                                  :name,
+                                                  :name,
                                                   label: { text: 'Other age range', size: 's' }),
             html_attributes: { "data-show-all-values" => true, } ) %>
 
         <% end %>
       <% end %>
 
-      <%= f.govuk_date_field(:programme_start_date, 
-                             date_of_birth: false, 
-                             legend: { text: 'Programme start date', size: 's' }, 
+      <%= f.govuk_date_field(:programme_start_date,
+                             date_of_birth: false,
+                             legend: { text: 'Programme start date', size: 's' },
                              hint: { text: 'For example, 2 11 2020' } ) %>
 
+       <%= f.govuk_date_field(:programme_end_date,
+                              date_of_birth: false,
+                              legend: { text: 'Programme end date', size: 's' },
+                              hint: { text: 'For example, 2 11 2020' } ) %>
```

Comment:
Hint text should be a year later than start date i.e. "For example, 2 12 2020"

Reply:
Thanks. Fixed this, although Ed gave me a different date to use.